src/main/java/svc/data/municipal/CourtDAO.java

```diff
@@ -9,23 +9,22 @@
 import org.springframework.jdbc.core.RowMapper;
 import org.springframework.stereotype.Repository;
 
+import svc.controllers.NotFoundException;
 import svc.data.jdbc.BaseJdbcDao;
 import svc.logging.LogSystem;
 import svc.models.Court;
 
 @Repository
 public class CourtDAO extends BaseJdbcDao {
-	public Court getByCourtId(Long courtId) {
-		try {
+	public Court getByCourtId(Long courtId) throws NotFoundException {
```

Comment:
No controller level exception should be done here. Controller(API level) exceptions should only be accessible at the controller layer. Down here in DAO land, catch your exception and return null. Then, at the controller, look for null and throw your exception.